.rubocop_todo.yml

```diff
@@ -0,0 +1,17 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-11-17 15:49:36 +1100 using RuboCop version 0.51.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+Lint/RescueWithoutErrorClass:
+  Exclude:
+    - 'lib/scraped/response/decorator/clean_urls.rb'
```

Comment:
This override is now in the 'base' rubocop file that is inherited from everypolitician-data, so isn't strictly necessary here (my guess is that `rubocop` didn't pick that up as you have a cached `rm .rubocop-https---raw-githubusercontent-com-everypolitician-everypolitician-data-master--rubocop-base-yml` from previous runs)

but it doesn't do any real harm to have here, and directs our attention to probably wanting to clean it up at a later point anyway,.